Procfile

```diff
@@ -0,0 +1,5 @@
+# Foreman Procfile. Start all dev server processes with: `foreman start`
+
+rails:      bundle exec rails s -p 3000
+webpack:    ./bin/webpack-dev-server
+sidekiq:    bundle exec sidekiq -q mailers -q default
```

Comment:
Wondering what devs think about running Sidekiq by default when starting the rails server locally? 

Maybe it's not needed? I'd say 99% of the time I don't need the jobs queue in dev, which makes me think it doesn;t need to run by default.

On the other hand it doesn't really do any _harm_ to run it, and it means any delayed emails will actually get "delivered" (and open in the browser with letter-opener).

Reply:
I've found it helpful with the current issue I'm working on, so I've gotten in the habit of starting it any time I start fresh even if I don't know for sure that I'll need it. It's handy to know when emails get sent and might help at some point to notice a change in behavior if they don't go out. I'd vote to have it start by default.

Reply:
To me, this looks almost like an exact duplicate of https://github.com/openfoodfoundation/openfoodnetwork/blob/master/docker-compose.yml. If I'm the only one using it, we better drop support for it. No point in maintaining something soon no one will use.